src/rpm/builder.rs

```diff
@@ -439,20 +444,20 @@ impl RPMBuilder {
             file_flags.push(entry.flag);
             file_usernames.push(entry.user.to_owned());
             file_groupnames.push(entry.group.to_owned());
-            file_inodes.push(ino_index as i32);
+            file_inodes.push(ino_index);
             file_langs.push("".to_string());
             let index = self
                 .directories
                 .iter()
                 .position(|d| d == &entry.dir)
                 .unwrap();
-            dir_indixes.push(index as i32);
+            dir_indixes.push(index as u32);
             base_names.push(entry.base_name.to_owned());
-            file_verify_flags.push(-1);
+            file_verify_flags.push(u32::MAX); // TODO: convert to bitfield or something so that we can use actual flag semantics
```

Comment:
https://github.com/rpm-rs/rpm/issues/52

Reply:
Could you reference the issue within the TODO comment?